Update netdata-charts.md (#17888)
netdata/netdata: docs/dashboards-and-charts/netdata-charts.md

## Title bar

> When you start interacting with a chart, you'll notice valuable information on the title bar:

Author: ```suggestion
When you start interacting with a chart, you'll notice valuable information on the Title bar:
```

<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>

> The Title bar elements:

Author: ```suggestion
The Title bar elements are:
```

> The Title bar elements are:

Reviewer: ```suggestion
The Title bar elements:
```

> The elements that you can find on this title bar are:

Author: ```suggestion
The Title bar elements are:
```

> The elements that you can find on this title bar are:

Author: ```suggestion
The elements that you can find on this Title bar are:
```

> When you start interacting with a chart, you'll notice valuable information on the top bar:

Author: ```suggestion
When you start interacting with a chart, you'll notice valuable information on the title bar:
```

<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>

> The elements that you can find on this title bar are:

Reviewer: ```suggestion
Top bar elements:
```

We call it the top bar (the sentence above), why "title" bar here

Author: Let me check!

Author: I will replace top bar above, sounds better as "title bar"

- **Netdata icon**: this indicates that data is continuously being updated, this happens if [Time controls](/docs/dashboards-and-charts/visualization-date-and-time-controls.md#time-controls) are in Play or Force Play mode.
- **Chart title**: on the chart title you can see the title together with the metric being displayed, as well as the unit of measurement.
- **Chart status icon**: possible values are: Loading, Timeout, Error or No data, otherwise this icon is not shown.

Along with viewing chart type, context and units, on this bar you have access to immediate actions over the chart:

<img src="https://github.com/netdata/netdata/assets/70198089/d21f326e-065c-4a08-bee9-69ad23736e38" width="200" />

- **Manage Alerts**: manage [Alert configurations](/docs/dashboards-and-charts/alerts-tab.md#alert-configurations-tab) for this chart.
- **Chart info**: get more information relevant to the chart you are interacting with.
- **Chart type**: change the chart type from **line**, **stacked**, **area**, **stacked bar** and **multi bar**.
- **Enter fullscreen mode**: expand the current chart to the full size of your screen.

> - **User settings**: save your settings for the chart at hand, so it persists across dashboard reloads.
>   - Personal has the top priority.
>   - Room and Space wide settings for a chart are shared across all users that don't have personal setting for it. 
> - **Drag and Drop the chart to a Dashboard**: add the chart to an existing custom [Dashboard](/docs/dashboards-and-charts/dashboards-tab.md) or directly create a new one that includes the chart.

Reviewer: ```suggestion
- **User settings**: save your settings for the chart at hand, so it persists across dashboard reloads.
  - Personal has the top priority.
  - Room and Space settings for a chart are shared across all users who don't have personal settings for it. 
- **Drag and Drop the chart to a Dashboard**: add the chart to an existing custom [Dashboard](/docs/dashboards-and-charts/dashboards-tab.md) or directly create a new one that includes the chart.
```
